recipes/hmmer/meta.yaml

```diff
@@ -17,7 +17,22 @@ requirements:
 
 test:
   commands:
-    - alimask -h 2>&1 | grep Usage > /dev/null
+    - alimask -h 2>&1 | grep "HMMER 3\." > /dev/null
+    - hmmalign -h 2>&1 | grep "HMMER 3\." > /dev/null
+    - hmmbuild -h 2>&1 | grep "HMMER 3\." > /dev/null
+    - hmmconvert -h 2>&1 | grep "HMMER 3\." > /dev/null
+    - hmmemit -h 2>&1 | grep "HMMER 3\." > /dev/null
+    - hmmfetch -h 2>&1 | grep "3\."
```

Comment:
Missing `> /dev/null` redirect?

Reply:
Deliberate as this tool's output is strangely different lacking the "HMMER 3..." text.